README.md

```diff
@@ -4,60 +4,96 @@
 
 ## Todo`s
 - [ ] check labels and selectors if they all are correct
-- [ ] configure NGINX from yaml
-- [ ] configure Let's Encrypt cert-manager from yaml
-- [ ] configure ingress form yaml
+- [ ] configure NGINX from yml
+- [ ] configure Let's Encrypt cert-manager from yml
+- [ ] configure ingress form yml
 - [ ] configure persistent & shared storage between nodes
-- [ ] reproduce setup locally
+- [x] reproduce setup locally
 
-> The dummy directory has some lb configurations that did not work properly on Digital Ocean but could be used as a starting point for getting it right
 
 ## Install Minikube, kubectl
 There are many Kubernetes distributions, but if you're just getting started, Minikube is a tool that you can use to get your feet wet.
 
 [Install Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
 
+# Open minikube dashboard
+```
+$ minikube dashboard
+```
+This will give you an overview.
+Some of the steps below need some timing to make ressources available to other
+dependent deployments. Keeping an eye on the dashboard is a great way to check
+that.
+
 ## Create a namespace locally
 ```shell
-kubectl create -f namespace-staging.json
+$ kubectl create -f namespace-staging.yml
 ```
+Switch to the namespace `staging` in your kubernetes dashboard.
 
-## Apply the config map to staging namespace
+## Setup config maps
 ```shell
-cd ./staging
-kubectl apply -f neo4j-configmap.yaml -f backend-configmap.yaml -f web-configmap.yaml
+$ cp db-migration-worker.template.yml config/db-migration-worker.yml
+# edit all variables according to the setup of the remote legacy server
+
+$ kubectl apply -f config/
 ```
 
 ## Setup secrets and deploy themn
+If you want to edit secrets, you have to `base64` encode them. See [kubernetes
+documentation](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret-manually).
 ```shell
-cd ./staging
-cp secrets.yaml.template secrets.yaml
-# change all vars as needed and deploy it afterwards
-kubectl apply -f secrets.yaml
+# example how to base64 a string:
+$ echo -n 'admin' | base64
```

Comment:
Why?

Reply:
Secrets are expected to be base64 encoded and get decoded. If you add a non-base64 encoded string, you get jibberish output for the environment variable in the pod